pkg/clustertree/cluster-manager/utils/leaf_model_handler.go

```diff
@@ -217,6 +217,7 @@ func createNode(ctx context.Context, clientset kubernetes.Interface, clusterName
 		if nodeAnnotations == nil {
 			nodeAnnotations = make(map[string]string, 1)
 		}
+		nodeAnnotations[nodeMode] = "one2cluster"
 		nodeAnnotations[utils.KosmosNodeOwnedByClusterAnnotations] = clusterName
```

Comment:
Does this mean that whether it's one2one or all mode, the "one2cluster" annotation will be added?